index.html

```diff
@@ -213,14 +228,28 @@ <h2>Resulting <code>custom.py</code></h2>
             enabled: true,
             default: true,
           },
-          // 'lightmapper_rd' isn't included as it's editor-only.
+          // 'lightmapper_cpu' (3.x only) isn't included as it's editor-only.
+          {
+            id: 'lightmapper_rd',
+            name: 'LightmapperRD',
+            description: 'Required to use LightmapperRD (4.x only).',
+            enabled: true,
+            default: true,
+          },
```

Comment:
I thought LightmapperRD was editor-only. To my knowledge, Godot doesn't support baking lightmaps at run-time at all, and I don't remember if UV2 wrapping is still exposed.

It's certainly a viable usecase in some scenarios now that lightmapping is GPU-accelerated, but you'd need a way to cache the UV2 and lightmaps to make subsequent map loads faster.